### PR TITLE
Add 1.20.0 changelog

### PR DESCRIPTION
**Pull request by delvh**

### CODE REVIEW

**Comment by lunny** [comment]
Is this a front port? Maybe we should send it to v1.20 branch first?

**Comment by delvh** [comment]
The other way around.
What difference does it make?

**Review by denyskon — APPROVED** [review]
I agree with @delvh , it's easier this way. Let's finally get this release done.

**Review by lunny — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,621 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.20.0](https://github.com/go-gitea/gitea/releases/tag/v1.20.0) - 2023-07-15
```
Comment (yardenshoham) [inline CHANGELOG.md]:
```suggestion
## [1.20.0](https://github.com/go-gitea/gitea/releases/tag/v1.20.0) - 2023-07-16
```

**Review by yardenshoham — APPROVED** [review]

**Pull request merged**